bin/compile

```diff
@@ -1,21 +1,41 @@
 #!/usr/bin/env bash
 
-echo "-----> DatadogBuildpack/compile"
-
 
 BIN_DIR=$(cd $(dirname $0); pwd)
 ROOT_DIR=$(dirname ${BIN_DIR})
 BUILD_DIR=$1
 CACHE_DIR=$2
-ENV_DIR=$3
+DEPS_DIR=$3
+DEPS_IDX=$4
 DATADOG_DIR="${BUILD_DIR}/.datadog"
+BUILDPACK_VERSION="$(cat "${ROOT_DIR}/VERSION")"
+
+echo "-----> DatadogBuildpack/$(basename $0) version $BUILDPACK_VERSION"
 
 mkdir -p "${DATADOG_DIR}/scripts"
 cp "${ROOT_DIR}/lib/scripts/update_agent_config.sh" "${DATADOG_DIR}/scripts/update_agent_config.sh"
 chmod +x "${DATADOG_DIR}/scripts/update_agent_config.sh"
 cp "${ROOT_DIR}/lib/scripts/get_tags.rb" "${DATADOG_DIR}/scripts/get_tags.rb"
 
 
+RUBY_CMD=ruby
+
+if ! which ruby > /dev/null; then
```

Comment:
```suggestion
if ! which $RUBY_CMD > /dev/null; then
```
nit

Reply:
Any reason? we're testing if ruby is installed/available so..

Reply:
I guess just that it's set to `ruby` now. But I do see if we end up changing the default setting of the variable, it could be a problem. Up to you whether we change it or not!